iOS_SDK/OneSignalSDK/UnitTests/UnitTestCommonMethods.m

```diff
@@ -272,6 +272,17 @@ + (void)backgroundApp {
     }
 }
 
++ (void)setAppInactive {
+    UIApplicationOverrider.currentUIApplicationState = UIApplicationStateInactive;
+}
+
++ (void)pullDownNotificationCenter {
```

Comment:
Weird, so simply just pulling down the notification center actually backgrounds then foregrounds then *backgrounds again*?

Reply:
I like the fact this was added as a helper method to keep the tests clean

Reply:
Thats what I found in my testing. I put break points in the lifecycle methods when pulling it down to see what happens it resigns active becomes active and then resigns active

Reply:
Very weird, what version of iOS? It is good to have the exact behavior in test so we should keep it, however just makes me wonder if this is some recent bug Apple introduced to iOS.